libvmaf/src/predict.c

```diff
@@ -287,9 +288,11 @@ int vmaf_predict_score_at_index(VmafModel *model,
                                                index);
 
         if (err) {
+            if (!propagate_metadata) {
```

Comment:
I'd like to understand this better - are there plans to make more changes to this function based on this flag, or will it just be used to suppress logs in case of error?

Reply:
In my case I used to suppress the logs because when calculating the `vmaf` score, each feature appends to the collector and attempts to access non-initialized features and is causing numerous errors. However, I am aware that this flag prevents visibility of 'real' errors, I am open to discuss that.